logger.go

```diff
@@ -77,3 +79,13 @@ func (l *Logger) Error(v any) {
 func (l *Logger) Errorf(format string, v ...any) {
 	l.lg.Errorf(l.ctx, format, v...)
 }
+
+// AddAttributes adds attributes to include in middleware-driven logs
+func (l *Logger) AddAttributes(attrbs map[string]any) {
```

Comment:
I am struggling with this interface. It doens't seem like it would be easy to use...

It feels like we will most likely be adding a single attribute at a time and not multiple... Maybe I am not thinking of this clearly.

Buit being forced to do a map literal to add a single attribute won't be so nice to use:
```
logger.Req(r).AddAttributes(map[string]any{"SessionID": sessionID})
```

If we think adding multiple will be needed, I think we should support both:
```
logger.Req(r).AddAttribute("SessionID", sessionID)
```

Reply:
Great points! Per our discussion, I've update the interface to accept a single attribute at a time.